src/controllers/userController/createNewUser.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from 'express';
+import { User } from '../../models/user';
+import { CustomError } from '../../middlewares/globalErrorHandler';
+
+export const createNewUser = async (req: Request, res: Response) => {
+  const { userName, userEmail, firstName, lastName } = req.body;
+  try {
+    const user = await User.findOne({
+      where: {
+        userEmail,
+      },
+    });
+    if (user) {
+      console.log('user already exists>--------------------');
```

Comment:
It doesn't send the correct error and the app crashes.
Delete console!

Reply:
it crashes?

Reply:
When I try to add the same user again to the DB( if user exist), the app crashes.

Reply:
addressed in slack message